### PR TITLE
A round of doc fixes

### PR DESCRIPTION
**Pull request by wtam2018**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind failing-test
> /kind feature
> /kind flake
> /kind code-refactoring

**What does this PR do / why we need it**:

**Have you updated the necessary documentation?**

* [x] Documentation update is required by this PR.
* [x] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: docs/commands/README.md [hunk 1]
```diff
@@ -2,10 +2,23 @@
 
 KAM binaries can be downloaded [https://github.com/redhat-developer/kam/releases/latest](https://github.com/redhat-developer/kam/releases/latest)
```
Comment (chetan-rns) [inline docs/commands/README.md:3]: Should we get rid of the word "Pipelines" in the heading?

File: docs/commands/service/README.md [hunk 2]
```diff
@@ -10,17 +10,28 @@ This command will create an application for the new service if the target applic
...
+      --git-repo-url string                            GitOps repository e.g. https://github.com/organisation/repository
+  -h, --help                                           help for add
+      --image-repo string                              Image repository of the form <registry>/<username>/<repository> or <project>/<app> which is used to push newly built images
+      --image-repo-internal-registry-hostname string   Host-name for internal image registry e.g. docker-registry.default.svc.cluster.local:5000, used if you are pushing your images to the internal image registry (default "image-registry.openshift-image-registry.svc:5000")
+      --pipelines-folder string                        Folder path to retrieve manifest, eg. /test where manifest exists at /test/pipelines.yaml (default ".")
+      --sealed-secrets-ns string                       Namespace in which the Sealed Secrets operator is installed, automatically generated secrets are encrypted with this operator (default "kube-system")
+      --sealed-secrets-svc string                      Name of the Sealed Secrets services that encrypts secrets (default "sealed-secrets-controller")
```
Comment (chetan-rns) [inline docs/commands/service/README.md:32]: We must also update the default values for sealed secrets flags here.

File: docs/commands/version/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# KAM Version Command
+
+
+Prrint out the version of KAM CLI.
```
Comment (chetan-rns) [inline docs/commands/version/README.md:4]:
```suggestion
Print out the version of KAM CLI.
```

File: docs/commands/README.md (hunk 1 above)
Comment (wtam2018) [reply docs/commands/README.md:3]: I think so.   There are still quite a few occurrence of "pipelines".

File: docs/commands/service/README.md (hunk 2 above)
Comment (wtam2018) [reply docs/commands/service/README.md]: Updated

File: docs/commands/README.md (hunk 1 above)
Comment (wtam2018) [reply docs/commands/README.md:3]: Probably not in this PR.

File: docs/commands/README.md [hunk 4]
```diff
@@ -2,10 +2,23 @@
...
 
-* [kam bootstrap](./bootstrap)
-* [kam build](./build)
-* [kam service](./service)
-* [kam webhook](./webhook)
-* [kam environment](./environment)
+
+
+<medium><pre>
+Kubernetes Application Managemer (KAM) CLI tool to scaffold your GitOps repository
```
Comment (chetan-rns) [inline docs/commands/README.md:9]:
```suggestion
Kubernetes Application Manager (KAM) is a CLI tool to scaffold your GitOps repository
```

File: docs/journey/day1/README.md [hunk 5]
```diff
@@ -19,7 +19,7 @@ You need to have the following installed in the OCP 4.x cluster.
 
 And, you will need this.
 
-* Create your [GitOps repository](prerequisites/gitops_repo.md)
+* Create your [GitOps repository](prerequisites/gitops_repo.md) and leave it unitialized.
```
Comment (chetan-rns) [inline docs/journey/day1/README.md:22]:
```suggestion
* Create your [GitOps repository](prerequisites/gitops_repo.md) and leave it uninitialized.
```

File: docs/journey/day1/README.md [hunk 6]
```diff
@@ -369,7 +369,7 @@ being executed.
 
 ![PipelineRun doing a dry run of the configuration](img/pipelinerun-dryrun.png)
 
-This validates that the YAML can be applied, by executing a `kubectl apply --dry-run`.
+This validates that the YAML can be applied, by executing a `oc apply -k --dry-run`.
```
Comment (chetan-rns) [inline docs/journey/day1/README.md:372]: We're using both `oc` and `kubectl` in different places. Maybe we can agree on one tool and standardize in a different PR

File: docs/journey/day1/prerequisites/argocd.md [hunk 7]
```diff
@@ -1,14 +1,30 @@
-### ArgoCD Installation - with operator (Recommended)
+# ArgoCD Installation - with operator (Recommended)
 
+There are three steps in this installation
+* [create `argocd` namespace ](#Create-argocd-namespace)
+* [nstall ArgoCD Operator](#install-ArgoCD-Operator)
```
Comment (chetan-rns) [inline docs/journey/day1/prerequisites/argocd.md:5]:
```suggestion
* [install ArgoCD Operator](#install-ArgoCD-Operator)
```

**Comment by chetan-rns** [comment]
Other than a few minor typos. Everything looks good to me. I'll raise a separate PR to fix them

**Pull request merged**